src/clients/user-client/src/vite-env.d.ts

```
@@ -0,0 +1,12 @@
+// eslint-disable-next-line @typescript-eslint/no-empty-object-type
+interface ViteTypeOptions {
+  // By adding this line, you can make the type of ImportMetaEnv strict
+  // to disallow unknown keys.
+  // strictImportMetaEnv: unknown
+}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)